projects/libgit2/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER Nelson Elhage <nelhage@nelhage.com>
+RUN apt-get update && apt-get install -y make autoconf automake libtool cmake
+RUN git clone --depth 1 -b oss-fuzz https://github.com/libgit2/libgit2 libgit2
+WORKDIR libgit2
+
+COPY fuzz_download_refs.cc build.sh $SRC/
```

Comment:
fuzz_download_refs.cc, build.sh and corpora should be later moved to upstram repo. See https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md